pkg/virt-api/rest/definitions.go

```diff
@@ -293,6 +293,14 @@ func NamespaceParam(ws *restful.WebService) *restful.Parameter {
 	return ws.PathParameter("namespace", "Object name and auth scope, such as for teams and projects").Required(true)
 }
 
+func ForceParam(ws *restful.WebService) *restful.Parameter {
+	return ws.QueryParameter("force", "Flag for the forced actions like Restart and Delete").Required(false)
```

Comment:
I suggest rephrasing this. Perhaps "Force requested action. Applies to Restart and Delete"

Reply:
Removed the `ForceParam` now